libs/wlgen/wlgen/rta.py

```diff
@@ -43,11 +43,20 @@ class Phase(_Phase):
     pass
 
 class RTA(Workload):
+    """
+    Class for creating RT-App workloads
+    """
 
     def __init__(self,
                  target,
                  name,
                  calibration=None):
+        """
+        :param target: Devlib target to run workload on.
+        :param name: Human-readable name for the workload
```

Comment:
I will try to add full stops where I've missed them (not going to spend too much time worrying about this though TBH). I guess the more correct thing would actually be no full stop, because they aren't really sentences. Anyway consistency is nicer than correctness here.